README.md

```diff
@@ -249,7 +257,200 @@ AdyenCardComponent(
   },
 );
 ``` 
+<br>
+
+### Apple Pay Component with Sessions flow
+1. From your backend, make a [`sessions`](https://docs.adyen.com/api-explorer/Checkout/71/post/sessions) request. The response contains:
+- `sessionData`: the payment session data.
+- `id`: a unique identifier for the session data.
+- The request body.
+
+Pass these values to your app.
+
+2. In your Flutter app, create a `ApplePayComponentConfiguration` instance. It requires a `ApplePayConfiguration` which contains your merchant id and merchant name.
+
+```dart
+final ApplePayConfiguration applePayConfiguration = ApplePayConfiguration(
+  merchantId: MERCHANT_ID,
+  merchantName: MERCHANT_NAME,
+);
+
+final ApplePayComponentConfiguration applePayComponentConfiguration = ApplePayComponentConfiguration(
+  // Change the environment to live when you are ready to accept real payments.
+  environment: Environment.test
+  clientKey: CLIENT_KEY,
+  countryCode: COUNTRY_CODE,
+  amount: AMOUNT,
+  applePayConfiguration: applePayComponentConfiguration,
+);
+```
+3. Create a `SessionCheckout` by calling the `AdyenCheckout.session.create()` while passing the required properties:
+```dart
+final sessionCheckout = await AdyenCheckout.session.create(
+  sessionId: sessionResponse.id,
+  sessionData: sessionResponse.sessionData,
+  configuration: applePayComponentConfiguration,
+);
+```
+4. Get the Apple Pay payment method from the `sessionCheckout` object.
+5. Use the Apple Pay component widget:
```

Comment:
Use the Apple Pay component to handle the payment. 

I don't think we need to use widget.